### PR TITLE
[flash_ctrl,dv] tb refactoring

### PR DESCRIPTION
**Pull request by jdonjdon**
- removed 'most' of redundent seqs

Signed-off-by: Jaedon Kim <jdonjdon@google.com>

### CODE REVIEW

**Comment by jdonjdon** [comment]
@cindychip This is long term overdue.
I merged read_only, write_only and all others to rw_seq.
I will remove retired sequences in separate PR after this.

File: hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson [hunk 1]
```diff
@@ -276,18 +276,18 @@
...
       desc: '''
-            Randomly enable ECC for a randomly selected set of pages. Randomly corrupt a single bit
-            in the memory that is about to be read and ensure that the ECC works - the corrupted
-            bit should be fixed. Corrupt randomly either the data or the ECC bits. Randomly corrupt
-            2 bits in the same word and ensure that the read results in error. Ensure that pages
-            with ECC not enabled reads back corrupted data without any errors. Verify both types,
-            pre-scramble ECC(integrity ECC, 4-bits) and post-scramble ECC(reliability ECC, 8-bits).
-	    Test status and control ECC bits.
+            Run read / write test and inject double bit error randomly for read transactions. -
+            both direct and controller read -.
```
Comment (cindychip) [inline hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson:282]: could you tell me what does the ending `-` mean?

File: hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson [hunk 2]
```diff
@@ -276,18 +276,18 @@
...
+            Check op_status.err and err_code.rd_err are asserted for ctrl read and
+            tlul response error for host read.
+            Check fatal alert is adderted for reliability ecc errors (double bits) and
```
Comment (cindychip) [inline hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson:285]: adderted? I do not know what this word means :)

**Review by cindychip — APPROVED** [review]

File: hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson (hunk 1 above)
Comment (jdonjdon) [reply hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson:282]: It was em dash. I will make longer.

File: hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson (hunk 2 above)
Comment (jdonjdon) [reply hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson:285]: fixed typo.

**Pull request merged**